otelconfig/otelconfig.go

```diff
@@ -569,7 +569,13 @@ func (c *Config) getMetricsHeaders() map[string]string {
 
 func setupTracing(c *Config) (func() error, error) {
 	endpoint, insecure := c.getTracesEndpoint()
-	if !c.TracesEnabled || endpoint == "" {
+	var enabled bool
+	if c.TracesEnabled == nil {
+		enabled = true
+	} else {
+		enabled = *c.TracesEnabled
+	}
+	if !enabled || endpoint == "" {
 		c.Logger.Debugf("tracing is disabled by configuration: no endpoint set")
```

Comment:
I know this is how it was in the first place, but should  we consider disambiguating this (and the metrics) message? it implies that there's no endpoint when it might be that it's disabled.

Reply:
Oh yes I agree. I have a change locally that I forgot to push up. Will do that now

Reply:
Is this excessive? https://github.com/honeycombio/otel-config-go/pull/123/commits/3cdb14433fe9579b4e899688658001e0e90e849d

Reply:
Not in my estimation; that's good UX.